modules/virt-generalizing-windows-sysprep.adoc

```diff
@@ -10,6 +10,10 @@ Generalizing an image allows that image to remove all system-specific configurat
 
 Before generalizing the VM, you must ensure the `sysprep` tool cannot detect an answer file after the unattended Windows installation.
 
+.Prerequisites
+
+* A running Windows virtual machine with the QEMU guest agent installed.
```

Comment:
Would including a link to _[Installing the QEMU guest agent and VirtIO drivers](https://sabrinajess.github.io/previews/CNV22648/virt/virtual_machines/virt-installing-qemu-guest-agent.html)_ in the assembly make sense? Or would a user know what QEMU guest agent is?

Reply:
Not a bad idea - I'll add

Reply:
That would be problematic in a module! :-)